### PR TITLE
chore(web/applications): upgrade plannin-inspectorare/address-lookup …

### PR DESCRIPTION
**Pull request by lutamKC**
## Describe your changes
upgrade plannin-inspectorare/address-lookup from 1.0.5 to 1.0.6

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/applications/create/applicant/applications-create-applicant.controller.js [hunk 1]
```diff
@@ -251,8 +250,7 @@ export async function updateApplicationsCreateApplicantAddress(
 			}
 			case 'selectAddress': {
 				const { errors: serviceErrors, addressList } = await findAddressListByPostcode(postcode, {
-					maxResults: 50,
-					minMatch: 0.9
+					maxResults: 50
```
Comment (lutamKC) [inline apps/web/src/server/applications/create/applicant/applications-create-applicant.controller.js:253]: the `postcode` api does not have a minmatch parameter

**Review by francescobrenci-AR — APPROVED** [review]

**Pull request merged**